overlays/haskell.nix

```diff
@@ -276,9 +276,7 @@ final: prev: {
 
         # Function to call stackToNix
         callStackToNix = import ../lib/call-stack-to-nix.nix {
-            pkgs = final.evalPackages.pkgs;
-            inherit (final.evalPackages.pkgs) runCommand;
-            inherit (final.evalPackages.haskell-nix) mkCacheFile materialize haskellLib;
+            pkgs = final;
```

Comment:
suspicious?

Reply:
I guess this does all line up: you pass in `pkgs` which will be used only to get the `evalPkgs`.